packages/account/src/Sections/Security/SelfExclusion/self-exclusion.jsx

```diff
@@ -9,18 +8,4 @@ const SelfExclusion = props => {
     return <SelfExclusionComponent is_appstore={is_appstore} {...props} />;
 };
 
-export default connect(({ client, ui }) => ({
```

Comment:
I guess they passed all of these as a prop to `SelfExclusionComponent` component, by removing all of these we may break the component, please check this one more time :pray: